File: src/main/java/fm/comunas/ddd/validation/model/impl/AggregatePartValidationImpl.java
package fm.comunas.ddd.validation.model.impl;

import fm.comunas.ddd.validation.model.AggregatePartValidation;
import fm.comunas.ddd.validation.model.enums.CodeValidationLevel;
import lombok.Builder;
import lombok.Data;

@Data
@Builder
public class AggregatePartValidationImpl implements AggregatePartValidation {

	private Integer seqNr;
	private CodeValidationLevel validationLevel;
	private String validation;

}

File: fm-dddrive/src/main/java/dddrive/app/ddd/model/impl/AggregatePartValidationImpl.kt
package dddrive.app.ddd.model.impl

import dddrive.app.ddd.model.AggregatePartValidation
import dddrive.app.ddd.model.enums.CodeValidationLevel

class AggregatePartValidationImpl(
	override val seqNr: Int,
	override val validationLevel: CodeValidationLevel,
	override val message: String,
	override val path: String? = null,
) : AggregatePartValidation
